services/http_gate/cfg/config.yml

```diff
@@ -0,0 +1,20 @@
+logger:
+  level: debug
+  
+rebalance_timer: 5m  # Interval to check nodes health
+
+connect_timeout: 60s  # Timeout to dial node
+
+request_timeout: 300s  # Timeout to check node health during rebalance
```

Comment:
Let's set the same as for `connect_timeout`

Reply:
I think @carpawell was just copying ENV params. I propose to create an issue and fix actual values in separate PR.

Reply:
yeah, did not change anything, tried to just make that work. suggest making any value changes in a separate PR

Reply:
https://github.com/TrueCloudLab/frostfs-dev-env/issues/5